apps/openmw/mwinput/inputmanagerimp.cpp

```diff
@@ -1022,6 +1022,7 @@ namespace MWInput
             return;
 
         if(MWBase::Environment::get().getWindowManager()->getMode() != MWGui::GM_Journal
+                && MWBase::Environment::get().getWindowManager()->getMode() != MWGui::GM_MainMenu
                 && MWBase::Environment::get().getWindowManager ()->getJournalAllowed())
```

Comment:
Wouldn't you want to move the Main Menu check into the getJournalAllowed method?